src/Stratis.Bitcoin.IntegrationTests/ProofOfStakeSteps.cs

```diff
@@ -46,6 +51,13 @@ public void PremineNodeWithWallet()
             this.PremineNodeWithCoins.Start();
         }
 
+        public void PremineNodeWithWalletWithOverrides()
+        {
+            var configParameters = new NodeConfigParameters { { "savetrxhex", "true" } };
+            this.PremineNodeWithCoins = this.nodeBuilder.CreateStratisCustomPosNode(KnownNetworks.StratisRegTest, configParameters).NotInIBD().WithWallet();
```

Comment:
@thefazzer can you please change `KnownNetworks.StratisRegTest` to just `new StratisRegTest`? we want to start moving away from the static container.